crates/tauri/src/plugin.rs

```diff
@@ -328,13 +356,45 @@ impl<R: Runtime, C: DeserializeOwned> Builder<R, C> {
   ///
   /// fn init<R: Runtime>() -> TauriPlugin<R> {
   ///   Builder::new("example")
-  ///     .js_init_script(INIT_SCRIPT.to_string())
+  ///     .js_init_script(INIT_SCRIPT)
   ///     .build()
   /// }
   /// ```
   #[must_use]
-  pub fn js_init_script(mut self, js_init_script: String) -> Self {
-    self.js_init_script = Some(js_init_script);
+  pub fn js_init_script(mut self, js_init_script: impl Into<String>) -> Self {
```

Comment:
```suggestion
  // TODO: Rename to initialization_script in v3
  pub fn js_init_script(mut self, js_init_script: impl Into<String>) -> Self {
```

we'd may have to rename the trait method but what do you think about this? imo it's so weird that we have js_init_script (plugin) and initialization_script (webview), both of which doing the same thing.

Reply:
Yep, makes sense to me